### PR TITLE
Add //= to syntax.md

### PR DESCRIPTION
**Pull request by deviaze**
Update syntax.md to include //= operator: 
- Add //= to compound assignment list,
- Add //= usage example to floordiv section.

### CODE REVIEW

File: _pages/syntax.md [hunk 1]
```diff
@@ -272,10 +272,18 @@ This restriction is made to prevent developers using other programming languages
...
 
 For numbers, `a // b` is equal to `math.floor(a / b)`; when `b` is 0, `a // b` results in infinity or NaN as appropriate.
+
+Compound assignments are supported with `//=`: 
+
+```lua
+local a = 4.2
+a //= 2
+print(a) --> 2
+```
```
Comment (alexmccord) [inline _pages/syntax.md:289]: Not sure we need an example for this.

File: _pages/syntax.md [hunk 2]
```diff
@@ -272,10 +272,18 @@ This restriction is made to prevent developers using other programming languages
 
 Luau currently does not support backtick string literals as a type annotation, so `` type Foo = `Foo` `` is invalid.
 
-Function calls with a backtick string literal without parenthesis is not supported, so `` print`hello` `` is invalid.
+Function calls with a backtick string literal without parethesis is not supported, so `` print`hello` `` is invalid.
```
[the remark this answers is not in the document]
Comment (deviaze) [reply _pages/syntax.md:275]: Ahhh! I tried redoing my fork/commit to get that `n` back there and guess it didn't work!! 

The subject-verb agreement was already incorrect in the current version, and there are a few other grammatical happenings on that page.

File: _pages/syntax.md (hunk 1 above)
Comment (deviaze) [reply _pages/syntax.md:289]: Fair, my other idea was including/namedropping that the `//=` exists a line earlier, and/or adding an example to the inline examples.

**Review by alexmccord — APPROVED** [review]
Looks great now! Thank you for your contribution.

**Comment by deviaze** [comment]
I just finished a pretty nice copyedit; I just committed it and realized this got approved. Could you also look at it and kinda add it to this one, or should I make a new PR? (I'm pretty new to PRs/etc.)

**Comment by alexmccord** [comment]
Has to be a new PR, sorry about that.

**Pull request merged**